--- ori.py      2016-05-29 01:30:26.004100753 +0000
+++ new.py      2016-05-29 01:32:49.612073233 +0000
@@ -7,6 +7,7 @@
 from django import forms
 from django.conf import settings
 from django.utils.translation import ugettext_lazy as _
+from django.core.files.storage import default_storage
 
 from ..core import utils
 from ..core.utils.markdown import Markdown
@@ -115,11 +116,17 @@
         media_path = os.path.join(settings.MEDIA_ROOT, upload_to)
         utils.mkdir_p(media_path)
 
-        with open(os.path.join(media_path, file.name), 'wb') as fh:
-            for c in file.chunks():
-                fh.write(c)
-
-            file.close()
+        with open(os.path.join(media_path, file.name), 'wb') as fh:
+            for c in file.chunks():
+                fh.write(c)
+
+            file.close()
+
+        fileRemote = default_storage.open(upload_to + '/' + file.name, 'wb')
+        fh = open(os.path.join(media_path, file.name), 'rb')
+        fileRemote.write(fh.read())
+        fileRemote.close()
+        fh.close()
 
         return file
 
